homeworks/05-redux-basic/src/store.ts

```diff
@@ -0,0 +1,15 @@
+import { createStore, compose } from "redux";
+
+import rootReducer from "./reducer";
+
+declare global {
+  interface Window {
+    __REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose;
+  }
+}
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const store = createStore(rootReducer, undefined, composeEnhancers());
```

Comment:
There should be a second homework part - Counter (increment, decrement) with combined reducer.

Reply:
Second part of homework was in codesandbox https://codesandbox.io/s/001-redux-forked-ghzsc